framework/src/interfaces/Coupleable.C

```diff
@@ -522,6 +598,22 @@ Coupleable::coupledMatrixTagValue(const std::string & var_name, TagID tag, unsig
   return var->matrixTagValue(tag);
 }
 
+const VariableValue &
+Coupleable::coupledMatrixTagValue(const std::string & var_name,
+                                  const std::string & tag_name,
+                                  unsigned int comp) const
+{
+  if (!_c_parameters.isParamValid(tag_name))
+    return coupledValue(var_name, comp);
```

Comment:
This is totally wrong. We do not have a default couple for matrix

Reply:
fixed.